server/chat-plugins/room-events.ts

```diff
@@ -55,30 +72,61 @@ export const commands: ChatCommands = {
 				return this.errorReply("You're missing a command parameter - see /help roomevents for this command's syntax.");
 			}
 
-			const eventNameActual = eventName.trim();
+			let eventNameActual = eventName.trim();
```

Comment:
I would just remove this line, you can see what i did in the other suggestions I made